src/components/FoodItem.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import './styles/FoodItem.css';
+
+class FoodItem extends Component {
+  state = {
+    qty: 0,
+  };
+
+  handleQty = (e) => {
+    this.setState({
+      qty: e.target.value,
+    });
```

Comment:
perfect how you handled having a state for information that will be changed inside the component.